<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="DESKTOP-EQA0C24" name="SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest" tests="6" failures="1" timestamp="07 Apr 2022 12:01:18 GMT" time="184.619" errors="0">
  <testcase name="createContact" time="20.419" classname="SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest.createContact(CreateContactwithBaseclassTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createContact -->
  <testcase name="createContact" time="51.810" classname="SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase name="createContact" time="25.797" classname="SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase name="createContact" time="28.228" classname="SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase name="createContact" time="34.091" classname="SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase name="createContact" time="24.274" classname="SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest">
    <skipped/>
  </testcase> <!-- createContact -->
</testsuite> <!-- SeleniumFrameworkWithTestNg.CreateContactwithBaseclassTest -->
